src/components/pegout/PegOutForm.vue

```diff
@@ -337,11 +348,10 @@ export default class PegOutForm extends Vue {
     return this.nextPage;
   }
 
-  mounted() {
-    const newUser = localStorage.getItem('ONBOARDED_USER_PEGOUT') !== 'true';
-    if (newUser) {
-      this.$tours.pegOutTour.start();
-    }
+  @Emit()
+  startVueTour() {
+    localStorage.setItem('ONBOARDED_USER_PEGIN', 'false');
+    this.$tours.pegOutTour.start();
```

Comment:
we should check the cookie value in the `mounted` hook to start the tour, otherwise it only be started with the user click